portfolio/src/main/java/com/google/sps/servlets/PaginationServlet.java

```diff
@@ -48,7 +49,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
         // Prepare information to be passed as a json
         long total = (long) allCommentsEntity.getProperty("total");
-        long max = (long) userInfoEntity.getProperty("max");
+        long max;
+        try {
+            max = (long) userInfoEntity.getProperty("max");
+        } catch (NullPointerException e) {
+            max = 10;
+        }
```

Comment:
```suggestion
        long max = (long) Optional.ofNullable(userInfoEntity.getProperty("max")).orElse(10);
```

Reply:
This makes a lot of sense. Thank you! I just added this as a new commit to this PR.